index.js

```diff
@@ -225,10 +240,12 @@ function getRunnersByTShirtSize(/* CODE HERE */) {
  * @param runners array of runners like the one inside the /data/runners.js file.
  * @returns a number which is the sum of the donations by all runners.
 */
-function tallyUpDonations(/* CODE HERE */) {
-  /* CODE HERE */
+function tallyUpDonations(runners) {
+  const totDonation =  runners.reduce((total,item) => { return total += item.donation;},0);
+  return totDonation;
 }
```

Comment:
I think for this problem the correct execution for line 244 would be `return total + item.donation`  The reason is because reduce automatically adds the next item to the total allowing the total to grow.  In the reduce method total is an accumulator.  `+=` is used when you want to say total = total + donations.  But that is already happening in the .reduce method.